src/main/scala/consul/v1/agent/service/ServiceRequests.scala

```diff
@@ -28,7 +28,9 @@ object ServiceRequests {
 
   private implicit lazy val CheckWrites: Writes[Check] = Json.writes[Check]
 
-  def apply(basePath: String)(implicit executionContext: ExecutionContext): ServiceRequests = new ServiceRequests{
+  def apply(basePath: String)(implicit executionContext: ExecutionContext, rb: ConsulRequestBasics): ServiceRequests = new ServiceRequests{
+
+    import rb._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/Codacy/scala-consul/file/1255759075/issues/source?bid=2788492&fileBranchId=2939115#l33)